commit ce30ada91951d0746879ae438da11f1ee8a90aa0
Author: Jia Tan <jiat0218@gmail.com>
Date:   Sun Oct 23 21:01:08 2022 +0800

    Tests: test_bcj_exact_size skips properly now if PowerPC filter disabled.

diff --git a/tests/test_bcj_exact_size.c b/tests/test_bcj_exact_size.c
index 0115ca60..551166cb 100644
--- a/tests/test_bcj_exact_size.c
+++ b/tests/test_bcj_exact_size.c
@@ -90,6 +90,9 @@ test_empty_block(void)
 #ifndef HAVE_DECODERS
 	assert_skip("Decoder support disabled");
 #else
+	if (!lzma_filter_decoder_is_supported(LZMA_FILTER_POWERPC))
+		assert_skip("PowerPC BCJ decoder is disabled");
+
 	// An empty file with one Block using PowerPC BCJ and LZMA2.
 	size_t in_size;
 	uint8_t *empty_bcj_lzma2 = tuktest_file_from_srcdir(